src/KafkaFlow.ApplicationInsights/AppInsightsConsumerEventsHandler.cs

```diff
@@ -22,33 +22,23 @@ public static Task OnConsumeError(IMessageContext eventContextMessageContext, Ex
         {
             {"topic" , eventContextMessageContext.ConsumerContext.Topic},
             {"partition" , eventContextMessageContext.ConsumerContext.Partition.ToString()},
-            {"offset" , eventContextMessageContext.ConsumerContext.Offset.ToString()},
         });
         eventContextMessageContext.Items.TryGetValue("timer", out var timer);
         var theTimer = (Stopwatch)timer;
         theTimer.Stop();
-        telemetryClient.TrackDependency("Kafka-Consumer",
-            eventContextMessageContext.ConsumerContext.Topic,
-            eventContextMessageContext.ConsumerContext.Partition.ToString(),
-            eventContextMessageContext.ConsumerContext.Offset.ToString(),
-            DateTimeOffset.UtcNow,
-            theTimer.Elapsed, "500", false);
+        telemetryClient.TrackKafkaDependency(eventContextMessageContext, "500", false, theTimer.Elapsed);
```

Comment:
Should we have specific error code for this? maybe we can start by the name of Exception type

Reply:
"500"

Reply:
fixed